Func.Isolated.Net7.With.AI/Program.cs

```diff
@@ -6,16 +6,12 @@
 using Func.Isolated.Net7.With.AI;
 
 var host = new HostBuilder()
-    .ConfigureFunctionsWorkerDefaults(builder =>
-    {
-        // Is added by package "Microsoft.Azure.Functions.Worker.ApplicationInsights".
-        // Documented here because it is still preview: https://github.com/Azure/azure-functions-dotnet-worker/pull/944#issue-1282987627
-        builder
-            .AddApplicationInsights()
-            .AddApplicationInsightsLogger();        
-    })
+    .ConfigureFunctionsWorkerDefaults()
     .ConfigureServices((ctx, serviceProvider) =>
     {
+        serviceProvider.AddApplicationInsightsTelemetryWorkerService();
+        serviceProvider.ConfigureFunctionsApplicationInsights();
```

Comment:
Do you have any documentation reference of these serviceProvider extensions?

Reply:
MS docs cover this, https://learn.microsoft.com/en-us/azure/azure-functions/dotnet-isolated-process-guide#application-insights

Reply:
Thanks. I see that https://learn.microsoft.com/en-us/azure/azure-functions/dotnet-isolated-process-guide#application-insights is mentioning this package reference as well